src/app/modules/consultations/response/response.component.html

```diff
@@ -10,12 +10,12 @@
             <p class="ministry-text m-0">{{profile?.consultation?.ministry?.name}}</p>
           </div>
         </div>
-        <div class="c-profile-card__middle-section">
+        <div  [ngStyle]="{style: expression}" class="c-profile-card__middle-section">
           <p class="m-0 deadline">{{'THE DEADLINE FOR THE CONSULTATION WAS' | translate}}
             {{profile?.consultation?.responseDeadline | date}}</p>
         </div>
 
-        <div class="c-profile-card__bottom-section">
+        <div class="c-profile-card__bottom-section" [(ngModel)]="model">
```

Comment:
@Harshwardhan18 Why is ngModel on this div?

Reply:
@Manzurkds I didn't add it. Let me check it and update it.

Reply:
@Manzurkds done